### PR TITLE
Added standard deviation and optional textual stat summary

### PR DESCRIPTION
**Pull request by jakobchristiaanse**

### CODE REVIEW

File: inflammation/views.py [hunk 1]
```diff
@@ -23,3 +23,14 @@ def visualize(data_dict):
...
+
+
+def stat_overview(data_dict):
+    """
+    Display textual overview of basic statistical properties of the inflammation data.
+
+    :param data_dict: Dictionary of name -> data to plot
+    """
+    print('Basic statistics of inflammation data:')
+    for key in data_dict:
```
Comment (MariovdBerg) [inline inflammation/views.py]: I suggest to change 'key' for a better name that represents what the key actually means.

**Review by MariovdBerg — CHANGES_REQUESTED** [review]
Nice work Jakob! This is a very structured and well written code. I can clearly see the course has done you good. Keep up the great work.

File: inflammation/views.py (hunk 1 above)
Comment (jakobchristiaanse) [reply inflammation/views.py]: That's a great idea, thanks Mario for your thorough review

Comment (jakobchristiaanse) [reply inflammation/views.py]: Fixed via commit 34fb497

**Review by MariovdBerg — APPROVED** [review]
I approve this message.

**Pull request merged**